drivers/video/video.c

```diff
@@ -123,6 +126,7 @@ struct video_type_inf_s
   struct v4l2_rect     clip;
   struct v4l2_fract    frame_interval;
   video_framebuff_t    bufinf;
+  FAR void             *vmem_heap;   /* for V4L2_MEMORY_MMAP buffers */
```

Comment:
```suggestion
  FAR uint8_t          *vmem_heap;   /* for V4L2_MEMORY_MMAP buffers */
```
help to compute the buffer location